### PR TITLE
chore: update doc for internal DOM utilities

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** N/A

## Summary

chore: update doc for internal DOM utilities

### CODE REVIEW

File: src/utils/dom.ts [hunk 1]
```diff
@@ -35,23 +53,50 @@ export function getModeName(el: HTMLElement): "light" | "dark" {
...
-export function getElementProp(el: Element, prop: string, fallbackValue: any): any {
-  const selector = `[${prop}]`;
+/**
+ * This helper returns the direction of a HTML element.
+ *
+ * @param {HTMLElement} el An element.
+ * @param {string} attribute An attribute name.
+ * @param {any} fallbackValue A fallback value.
+ * @returns {any} The value.
+ * @depecated
```
Comment (driskull) [inline src/utils/dom.ts:76]: deprecating this function as it shoudn't be used.

It actually doesn't return a prop, it returns an attribute.

File: src/utils/dom.ts [hunk 2]
```diff
@@ -35,23 +53,50 @@ export function getModeName(el: HTMLElement): "light" | "dark" {
...
-export function getElementProp(el: Element, prop: string, fallbackValue: any): any {
-  const selector = `[${prop}]`;
+/**
+ * This helper returns the value of an attribute on an element.
+ *
+ * @param {HTMLElement} el An element.
+ * @param {string} attribute An attribute name.
+ * @param {any} fallbackValue A fallback value.
+ * @returns {any} The value.
+ * @depecated
```
Comment (jcfranco) [inline src/utils/dom.ts:76]: Typo: `@depecated`

**Review by jcfranco — APPROVED** [review]
Awesome!

Sidebar: @benelan do you think we should enable [`require-hyphen-before-param-description`(autofixable 🎉)](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/require-hyphen-before-param-description.md#readme) to make it easier to scan?

File: src/utils/dom.ts (hunk 2 above)
Comment (driskull) [reply src/utils/dom.ts:76]: eek ⚔️ 🪦

**Pull request merged**